webapp-django/crashstats/signature/views.py

```diff
@@ -318,7 +318,10 @@ def signature_graphs(request, params, field):
 
 @pass_validated_params
 def signature_comments(request, params):
-    '''Return a list of non-empty comments. '''
+    """Return a list of non-empty comments."""
+    # Users can't see comments unless they have view_pii permissions.
+    if not request.user.has_perm('crashstats.view_pii'):
+        return http.HttpResponseForbidden()
```

Comment:
This check makes sure no one can see the data--it throws a 403. So even if the tab was showing, it wouldn't work and people can't figure out the url and hit it directly. Etc.